Header header
int16 cycleTime
int16 i2c_errors_count